### DISCUSSION TITLE
"failed to fetch" existing projects in Firebase

### DISCUSSION THREAD

**Discussion by talkd** [AntlerVC/firetable#932] [Support / Q&A] [answered]
Hi - I am really struggeling with my account in Rowy. It all worked fine initially. But I have configured my Google Cloud account and Flutterflow as well - which might be a reason to the issue. I am the owner of the organization and the project, and I use the e-mail related to this. 

I seem pretty stuck, and I am not able to reset/delete my account - so I can start all over? I tried to figure out what was wrong in Firebase / Google Cloud - but it might have messed it up.

![Failed to fetch](https://user-images.githubusercontent.com/116219872/197208420-2fb2a61b-3040-48ff-a5fe-6d67a93f7ba6.PNG)

**Comment by talkd** [comment]
Hoping for help soon. My project relies on access to Rowy🙏

**Comment by harinij** [comment, marked as answer]
Hi @talDigital We missed seeing this issue earlier :( If this is still an issue we would be happy to hop on a call to get it resolved for you. You can book a time with us [here](https://cal.com/shams/rowy).
